_includes/head.html

```diff
@@ -2,6 +2,17 @@
   <meta charset="utf-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1">
+  <!-- facebook -->
+  <meta property="og:title" content="Basement">
+  <meta property="og:description" content="Ground floor CSS utility classes">
+  <meta property="og:image" content="/assets/images/basement_facebook_share_card.jpg">
+  <meta property="og:url" content="https://basement.sanctuary.computer/">
+  <!-- twitter -->
+  <meta name="twitter:title" content="Basement">
+  <meta name="twitter:description" content="Ground floor CSS utility classes">
+  <meta name="twitter:image" content="/assets/images/basement_twitter_share_card.jpg">
```

Comment:
For social share cards, the URL to images (`twitter:image` or `og:image`) needs to be absolute.

Could you pls update to `https://basement.sanctuary.computer/assets/images/basement_twitter_share_card.jpg`